Sources/TuistCore/Models/CopyFilesAction.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import TSCBasic
+
+public struct CopyFilesAction: Equatable {
+    public var name: String
+    public var destination: Destination
+    public var subpath: String
+    public var files: [FileElement]
+
+    public enum Destination: UInt, Equatable {
+        case absolutePath = 0
```

Comment:
The fact that those values are the ones used in Xcode projects is not obvious. I'd rather have something like:

```swift
public enum Destination: Equatable {
  var xcodeprojValue: UInt {
    // switch 
  }
}
```